util/fetch_shas_TOOLS.txt

```diff
@@ -1 +1,3 @@
 rust
```

Comment:
When would 'rust' be used with this setup?

Reply:
I don't expect it to be used at all, actually, but I hesitated to remove it from the sha generation list. I think it might be a good idea to remove it sometime after this change is done.